cvat/apps/engine/serializers.py

```diff
@@ -392,6 +432,20 @@ def validate_chunk_size(self, value):
             raise serializers.ValidationError('Chunk size must be a positive integer')
         return value
 
+    def validate_job_file_mapping(self, value):
+        existing_files = set()
+
+        for job_files in value:
+            for filename in job_files:
+                if filename in existing_files:
+                    raise serializers.ValidationError(
+                        "The same file cannot be used in several jobs in the job file mapping"
```

Comment:
> This could also occur if the file is used twice in the same job.

Thanks, I'll correct the message.

> Also, it would be worth including the offending file name.

Thanks, it's a good idea.

Reply:
Done